hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -656,15 +656,29 @@
             <aws enabled="false">
                 <access-key>my-access-key</access-key>
                 <secret-key>my-secret-key</secret-key>
-                <iam-role>dummy</iam-role>
                 <region>us-west-1</region>
                 <host-header>ec2.amazonaws.com</host-header>
-                <security-group-name>hazelcast-sg</security-group-name>
+                <connection-timeout-seconds>7</connection-timeout-seconds>
```

Comment:
I see that there are also `aws`, `gcp` and `azure` elements under the `advanced-network/join` element. Should we also update them?

Reply:
I think we should update them too. However I am not sure if the config properties are different for that element. I will have to search a bit, then I will update them.